src/DebugEngineHost.Stub/DebugEngineHost.ref.cs

```diff
@@ -394,6 +394,16 @@ public static void FindNatvisInSolution(NatvisLoader loader)
             throw new NotImplementedException();
         }
 
+        /// <summary>
+        /// Searches the extension manager for natvis files, invoking the loader on any which are found.
+        /// </summary>
+        /// <param name="loader">Natvis loader method to invoke</param>
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Naming", "CA1704:IdentifiersShouldBeSpelledCorrectly", MessageId = "Natvis")]
+        public static void FindNatvisInVSIX(NatvisLoader loader)
```

Comment:
>FindNatvisInVSIX [](http://example.com/codeflow?start=27&length=16)

nit: How about `FindNatvisExtensions`?

Reply:
Actually, can we just change `FindNatvisInSolution` to `FindNatvis` and have it do both? `FindNatvis` MIEngine currently violates VS Threading rules and does all this stuff on a background thread which turns around and calls shell services. It would be good to just have one spot to fix instead of more than one.